utils_linux.go

```diff
@@ -196,11 +197,20 @@ func (r *runner) run(config *specs.Process) (int, error) {
 		r.destroy()
 		return -1, err
 	}
+
+	if len(r.listenFDs) > 0 && r.preserveFDs > 0 {
+		return -1, fmt.Errorf("Preserve FDs and socket activation (LISTEN_FDS) are mutually incompatible")
+	}
```

Comment:
This feels like a bit of a cop out, but I couldn't think of a better way. The only alternative I thought of was to specify that preserveFds preserves N over and above `${LISTEN_FDS:-0}` but that seemed pretty fugly. It seems unlikely to me that one would write an application which uses both socket activation and expects other random open fds to be present upon exec.